samples/sample_misc/wayland/src/class_wayland.cpp

```diff
@@ -347,20 +351,53 @@ struct wl_buffer * Wayland::CreatePrimeBuffer(uint32_t name
     , int32_t pitches[3])
 {
     struct wl_buffer * buffer = NULL;
-    if(NULL == m_drm)
-        return NULL;
 
-    buffer = wl_drm_create_prime_buffer(m_drm
-            , name
-            , width
-            , height
-            , format
-            , offsets[0]
-            , pitches[0]
-            , offsets[1]
-            , pitches[1]
-            , offsets[2]
-            , pitches[2]);
+#if defined(WAYLAND_LINUX_DMABUF_SUPPORT)
```

Comment:
Hm... Should we make this unconditional and require this feature upfront on configuration level? @dmitryermilov, @onabiull : how do you think?

Reply:
Nice to have it by default of course but I'm not sure about availability of wayland-scanner>=1.15, wayland-protocols>=1.15 is OS distributions.

Reply:
Not  blocker for merge, IMO